server/fishtest/templates/tests_run.mak

```diff
@@ -517,7 +527,7 @@
       if (btn.dataset.options) testOptions = btn.dataset.options;
 
       if (testOptions) {
-        const { tc, new_tc, threads, options, book, stop_rule, bounds, games, base_branch, base_signature } = JSON.parse(testOptions);
+        const { tc, new_tc, threads, options, book, stop_rule, bounds, games, test_branch, base_branch, test_signature, base_signature } = JSON.parse(testOptions);
```

Comment:
@dav1312 what are doing these new parameters?

Reply:
Set the test branch and the test signature...?

Reply:
I'm testing the PR on DEV but I don't know how to check these new parameters vs master. I need some help to understand the high level function they are used for (eg new test, reschedule, back button etc.) :)

Reply:
It's just for this
![image](https://user-images.githubusercontent.com/63931154/189484648-819ef44a-46ee-4b3c-9322-db972e01597f.png)

And if it is a reschedule pressing the PT presets won't change those values

Reply:
Thank you :)